recipe_scrapers/_abstract.py

```diff
@@ -109,6 +109,10 @@ def nutrients(self):
         """Nutrients of the recipe."""
         raise NotImplementedError("This should be implemented.")
 
+    def keywords(self):
```

Comment:
I wonder.. we need all of our field methods to be callable without any arguments provided, so that `to_json` can run.

However: we could allow default arguments.

Should we add a `limit=5` parameter, and only return that many keywords?

(if so we could consider something similar for categories / etc)

The idea would be to make it less likely that individual scrapers can emit results that are significantly larger / different to others.

Or perhaps it's better to wait until we receive any complaints/feedback about that as a particular problem before doing anything about it.

Reply:
I think it's better to leave unless there are complaints, _most_ sites only list a few regardless so it's not common that a long list is output with a few exceptions